### DISCUSSION TITLE
External converters and custom clusters

### DISCUSSION THREAD

**Discussion by grebenyuk-victor** [Koenkk/zigbee2mqtt#23562] [General]
When I use external converters, I have a problem with customs clusters

My device has one custom cluster 0xfc00 (64512) and two custom attributes 0x0a00 (2560), 0x0a01 (2561), manufacturercode 0x4c44
In frontend I cannot read values of this attributes due to error. But I can write attributes
```
debug: 	z2m: Received Zigbee message from 'S1', type 'readResponse', cluster 'manuSpecificAssaDoorLock', data '{"2560":0}' from endpoint 1 with groupID 0
debug: 	z2m: No converter available for 'My Device' with cluster 'manuSpecificAssaDoorLock' and type 'readResponse' and data '{"2560":0}'
```

As far as I understand, cluster 0xfc00 exists in `cluster.js` in herdsman library with name `manuSpecificAssaDoorLock` and has no 0x0a00/0x0a01 attributes. Furthermore, there is no manufacturer specific code in this cluster!

**Q1**: Why library processes the request with wrong manufacturer specific code? (my code is 0x4c44, and `manuSpecificAssaDoorLock` doesn't have any code)
**Q2**: How can I work with custom clusters with external converters without adding them to `cluster.js`?

Thanks for reply!

P.S. My code
```
...
modernExtend.enumLookup({
	name: 'switch_mode',
	lookup: {
		'toggle': 0,
		'alternate': 2,
	},
	endpointName: `button_${i + 1}`,
	cluster: 0xfc00,
	attribute: {
		ID: 0x0a00,
		type: 0x30,
	},
	description: 'Operation method of the switch',
	access: 'ALL',
	entityCategory: 'config',
	zigbeeCommandOptions: {
		manufacturerCode: 0x4c44,
	},
}),
...
```

**Comment by grebenyuk51** [comment]
I don't know how to delete discussion
see #23563
